lib/slayer/rspec.rb

```diff
@@ -33,6 +33,7 @@
     return "expected command not to have message: #{@message}" if !@message.nil? && result.message == @message
     return "expected command not to have value: #{@value}" if !@value.nil? && result.value == @value
     return "expected command not to have status :#{@status}" if !@status.nil? && result.status == @status
+
```

Comment:
Convenient spot for me to mention that we should have an extra chain method:

```rb
chain :with do |args|
  @status = args[:status] if args.include? :status
  @message = args[:message] if args.include? :message
  @value = args[:value] if args.include? :value
end
```

This would clean up the expectations:

```rb
it { is_expected.to be_failed_result.with(status: :no_status, message: "You Shall Not Pass!") }
```

Reply:
Good callouts on the matchers